hazelcast-enterprise/start-hazelcast.sh

```diff
@@ -22,8 +22,7 @@ if [ -n "${PROMETHEUS_PORT}" ]; then
 fi
 
 if [ -n "${LOGGING_LEVEL}" ]; then
-  sed -i "s/java.util.logging.ConsoleHandler.level = INFO/java.util.logging.ConsoleHandler.level = ${LOGGING_LEVEL}/g" logging.properties
-  sed -i "s/.level= INFO/.level= ${LOGGING_LEVEL}/g" logging.properties
+  sed -i "s/rootLogger.level=info/rootLogger.level=${LOGGING_LEVEL}/g" log4j2.properties
```

Comment:
Consider using variable substitution (instead of sed) as described here: https://dougbreaux.github.io/2013/06/05/log4j-log-level.html

Reply:
Done